lib/ansible/playbook/playbook_include.py

```diff
@@ -84,7 +84,8 @@ def load_data(self, ds, variable_manager=None, loader=None, basedir=None):
                 playbook = os.path.join(basedir, playbook)
 
             # might still be collection playbook
-            playbook_collection = _get_collection_name_from_path(playbook)
+            playbook_collection = _get_collection_name_from_path(playbook) or AnsibleCollectionConfig.default_collection
```

Comment:
This change is unnecessary. As None value of `playbook_collection` is already handled on line number 92